### PR TITLE
Importing the old repo into this one

### PR DESCRIPTION
**Pull request by jrgns**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+ruby "1.9.3", :engine => "jruby", :engine_version => "1.7.19"
```
Comment (suyograo) [inline Gemfile]: Why this line? can we remove it?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2012–2015 Elasticsearch <http://www.elastic.co>
```
Comment (suyograo) [inline LICENSE]: 2016 :)

File: lib/logstash/inputs/cloudwatch.rb [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+  config :combined, :validate => :boolean, :default => false
+
+  public
+  def aws_service_endpoint(region)
+    { region: region }
+  end
+
+  public
+  def register
+    require "aws-sdk"
```
Comment (suyograo) [inline lib/logstash/inputs/cloudwatch.rb]: can you push this require to the top?

**Comment by suyograo** [comment]
@jrgns can you add tests? Even if they are mocks should be ok

File: Gemfile (hunk 1 above)
Comment (jrgns) [inline Gemfile]: See this for an explanation: https://github.com/logstash-plugins/logstash-input-example/pull/3

**Comment by jrgns** [comment]
I've added some boilerplate for tests and a single test to check that the register method works.

**Comment by suyograo** [comment]
@jrgns thanks. I merged and bumped version to 1.1.0. Can you please add `dev_ops@elastic.co` to RubyGems.org owners so we can push this?

**Comment by jrgns** [comment]
@suyograo Done

Awesome, thanx!

**Pull request merged**